Runtime/Editor/ProjectSettings.cs

```diff
@@ -101,7 +101,7 @@ private void DrawGameSettings()
                 if (match.Success)
                 {
                     string regexKey = match.Value;
-                    Debug.LogWarning("You accidentally used the domain url instead of the domain key,\nWe took the domain key from the url.: " + regexKey);
+                    LootLockerLogger.Log("You accidentally used the domain url instead of the domain key,\nWe took the domain key from the url.: " + regexKey, LootLockerLogger.LogLevel.Info);
```

Comment:
```suggestion
                    LootLockerLogger.Log("You accidentally used the domain url instead of the domain key,\nWe took the domain key from the url.: " + regexKey, LootLockerLogger.LogLevel.Warning);
```

Reply:
Ignored